content/tutorials/r_gbif_name_matching/index.Rmd

```diff
@@ -12,70 +12,92 @@ output:
 
 ## Introduction
 
-This tutorial will explain how you can match a list of 'scientific names' to the GBIF taxonomic backbone
+This tutorial will explain how you can match a list of _scientific names_ against the [GBIF backbone taxonomy](https://www.gbif.org/dataset/d7dddbf4-2cf0-4f39-9b2a-bb099caae36c).
 
-Important is that you have [rgbif](https://github.com/ropensci/rgbif) and [inborutils](https://inbo.github.io/inborutils/) installed and available:
+Important is that you have the most recent version of [inborutils](https://inbo.github.io/inborutils/) installed and available:
 
-```{r message=FALSE, warning=FALSE}
-library(tidyverse) # tidyverse
-library(rgbif)     # To Match GBIF
-library(inborutils) # wrap GBIF api data
+```{r install_pkgs, eval=FALSE}
+remotes::install_github("inbo/inborutils")   # install inborutils
+```
+
+```{r load_libraries, message=FALSE, warning=FALSE}
+library(tidyverse)    # To do datascience
+library(rgbif)        # To lookup names in the GBIF backbone taxonomy
+library(inborutils)   # To wrap GBIF API data
 library(knitr)
 ```
 
 ## Read data file containing the scientific names
 
 Read file containing the scientific names you want to check against the [GBIF](https://www.gbif.org/species/search?q=) taxonomic backbone:
 
-```{r}
+```{r read_species_list}
 species_list <- read_csv("https://raw.githubusercontent.com/inbo/inbo-pyutils/master/gbif/gbif_name_match/sample.csv", trim_ws = TRUE, col_types = cols())
 ```
    
 Take a look at the data:
 
-```{r}
-knitr::kable(head(species_list))
+```{r preview_species_list}
+kable(species_list)
 ```
 
-## Request species information
+## Request taxonomic information
 
-Match the column containing the scientificName with GBIF, using the `gbif_species_name_match` function from the [inborutils](https://inbo.github.io/inborutils/reference/gbif_species_name_match.html) package:
+Match the column containing the scientific name with GBIF, using the `gbif_species_name_match` function from the [inborutils](https://inbo.github.io/inborutils/reference/gbif_species_name_match.html) package:
 
-```{r, message=FALSE, warning=FALSE}
+```{r use_function_first_time, message=FALSE, warning=FALSE}
```

Comment:
Maybe better to explain the first argument (df) and the second argument (name) before the code. The first piece of code can then be: 

````
species_list_matched <- gbif_species_name_match(df = species_list, name = "name")
````

and the second equivalent piece of code, using piping as in your `similar_code` chunk.

Reply:
You are right. I did it here: https://github.com/inbo/tutorials/pull/146/commits/ff7e6347e258049eec6384f56133a8d09eb9740b

I also renamed `species_list` input as `species_df` as we read a data.frame in and we use it as input for argument `df`.